src/systems/filecoin_nodes/clock/_index.md

```diff
@@ -39,14 +42,13 @@ Computer-grade clock crystals can be expected to have drift rates on the order o
   - See how [go-ethereum does this](https://github.com/ethereum/go-ethereum/blob/master/p2p/discv5/ntp.go) for inspiration
 - clients CAN consider using cesium clocks instead for accurate synchrony within larger mining operations
 
-Assuming a majority of rational participants, the above should lead to relatively low skew over time, with seldom more than 10-20% clock skew that should be rectified periodically by the network, as is the case in other networks. This assumption can be tested over time by ensuring that:
-
-- (real-time) epoch time is as dictated by the protocol
-- (historical) the current epoch number is as expected
+Assuming a majority of rational participants, the above should lead to relatively low skew over time, with seldom more than 10-20% clock skew that should be rectified periodically by the network, as is the case in other networks. This assumption will be maintained over time with both the real-time -- blocks with timestamps outside of current epoch time are rejected -- and historical -- blocks with timestamps that don't match protocol specified epochs are rejected -- validation rules.
 
 ## Future work
 
 If either of the above metrics show significant network skew over time, future versions of Filecoin may include potential timestamp/epoch correction periods at regular intervals.
 
+When recoverying from exceptional chain halting events (for example all implementations panic on a given block) the network will potentially opt for special purpose "dead zone" rules banning the authoring of blocks during the outage epochs to prevent attack vectors related to unmined epochs during chain restart.
```

Comment:
Would seek ack from all impls and @jbenet on this

Reply:
Its just a relevant idea, not a commitment (as go-filecoin repr I ack 👿 ).  Might be best to just nuke "future work" from the spec.

Reply:
Sorry I hadn't seen the header. You're all set. Thanks for pointing out.